src/web/app/components/question-responses/grq-rgq-view-responses/grq-rgq-view-responses.component.ts

```diff
@@ -174,6 +178,14 @@ export class GrqRgqViewResponsesComponent extends InstructorResponsesViewBase im
             this.responsesToShow[user] = this.responsesToShow[user] || {};
             this.responsesToShow[user][other] = this.responsesToShow[user][other] || [];
             this.responsesToShow[user][other].push(questionCopy2);
+
+            if (!this.userHasRealResponses[user]) {
+              const hasRealResponse: boolean =
```

Comment:
```suggestion
              this.userHasRealResponses[user] =
```

can be simplified :P